VideoLocker/src/main/java/org/edx/mobile/loader/CoursesAsyncLoader.java

```diff
@@ -52,6 +45,18 @@ public CoursesAsyncLoader(Context context) {
         return result;
     }
 
+    private List<EnrolledCoursesResponse> getEnrolledCourses() {
+        List<EnrolledCoursesResponse> ret = null;
+
+        try {
+            ret = environment.getServiceManager().getEnrolledCourses(false);
```

Comment:
@mdinino and I had an offline discussion. Looks like Retrofit 1.9 itself does not have cacheing, but  we should be able to cache at the okhttp layer. @mdinino will look into it more.

Seems like the issue here was that we need to be able to cache the response from Retrofit, where we used to be caching the response in the deprecated org.apache.http library.